packages/metadata-agent/src/index.js

```
@@ -0,0 +1,115 @@
+ * @type {Object}
+ *
+ * @example
+ * import { api } from '@lightspeedwp/metadata-agent';
+ *
+ * api.labelUtils.validate('type:bug');
+ * api.createClient({ token: process.env.GITHUB_TOKEN });
+ * api.validation.validateTier1(issues);
+ */
+export const api = {
+  version: VERSION,
+  labelUtils,
```

ESLint

[error] 'labelUtils' is not defined.

(no-undef)